XiEditor/AppDelegate.swift

```diff
@@ -14,14 +14,14 @@
 
 import Cocoa
 
-//TODO: preferred font should be a user preference
-let defaultFont = CTFontCreateWithName("InconsolataGo" as CFString?, 14, nil)
-
 @NSApplicationMain
 class AppDelegate: NSObject, NSApplicationDelegate {
 
     var dispatcher: Dispatcher?
 
+    //TODO: preferred font should be a user preference
+    static let defaultFont = CTFontCreateWithName("InconsolataGo" as CFString?, 14, nil)
```

Comment:
heh if we're going to put this in app delegate then it should be an instance variable, because app delegate is already a singleton. Sorry if I was unclear 😂